src/test/java/com/github/kokorin/jaffree/ffmpeg/FFmpegTest.java

```diff
@@ -392,6 +392,21 @@ public void consume(Frame frame) {
         Assert.assertNotNull(result);
     }
 
+    @Test
+    public void testUserAgent() throws Exception{
+      FFmpegResult result;
+      String URL = "https://static.videezy.com/system/protected/files/000/007/213/Biking_Girl_Alpha.mov?md5=zJB3WS6tzcdWmKjzHnSTLA&expires=1553233302";
```

Comment:
3rd party resources should not be used as it may affect test stability

Reply:
Resolved with adding a testing HTTP Server